privacyidea/lib/tokens/u2ftoken.py

```diff
@@ -188,10 +188,10 @@
 #
 # The solokeys image is copyright (C) 2020 Solokeys. License: CC-BY-SA 4.0
 #
-IMAGES = {"yubico": "static/img/FIDO-U2F-Security-Key-444x444.png",
-          "plug-up": "static/img/plugup.jpg",
-          "u2fzero.com": "static/img/u2fzero.png",
-          "solokeys": "static/img/solokeys.png"}
+IMAGES = {"yubico": "privacyidea/static/img/FIDO-U2F-Security-Key-444x444.png",
+          "plug-up": "privacyidea/static/img/plugup.jpg",
+          "u2fzero.com": "privacyidea/static/img/u2fzero.png",
+          "solokeys": "privacyidea/static/img/solokeys.png"}
```

Comment:
maybe we should add a comment that these are filesystem paths, not URIs, if someone wants to add/change an image.

Reply:
Done in  c03cd67a